msgs/confirmSigMsg.go

```diff
@@ -0,0 +1,74 @@
+package msgs
+
+import (
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/FourthState/plasma-mvp-sidechain/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+const (
+	// ConfirmSigMsgRoute is used for routing this message.
+	ConfirmSigMsgRoute = "confirmSig"
+)
+
+// SpendMsg implements the RLP interface through `Transaction`
+type ConfirmSigMsg struct {
+	Input1    plasma.Input
+	Input2    plasma.Input
+}
+
+// Type implements the sdk.Msg interface.
+func (msg ConfirmSigMsg) Type() string { return "confirm_sig" }
+
+// Route implements the sdk.Msg interface.
+func (msg ConfirmSigMsg) Route() string { return ConfirmSigMsgRoute }
+
+// GetSigners will attempt to retrieve the signers of the message.
+// CONTRACT: a nil slice is returned if recovery fails
+func (msg ConfirmSigMsg) GetSigners() []sdk.AccAddress {
+	positionHash1 := utils.ToEthSignedMessageHash(msg.Input1.Position.Bytes())
```

Comment:
this is incorrect. This is saying the signature is over the position which is wrong. 

From what I remember, the message is expected to have not actual signers. GetSigners should just return nil